content/guides/tooling/typescript-support.md

```diff
@@ -152,19 +152,19 @@ instructions.
 
 ### Types for plugins
 
-You can utilize Cypress's type declarations in your
-[plugins file](/guides/tooling/plugins-guide) by annotating it like the
-following:
+You can utilize Cypress's type declarations by annotating the
```

Comment:
Please revert changes to this file, I'm going to make a few edits to it separately!